spec/workers/emails/remove_old_emails_worker_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+RSpec.describe Emails::RemoveOldEmailsWorker, type: :woker do
+  include_examples "#enqueues_on_correct_queue", "low_priority"
+
+  describe "#perform" do
+    let(:worker) { subject }
+
+    it "fast destroys notifications" do
+      allow(EmailMessage).to receive(:fast_destroy_old_retained_email_deliveries)
```

Comment:
I'm not seeing tests of the SQL command, and am wondering if letting this `worker.perform` call the method would be an adequate verification of the SQL command?